src/functions-reference/real-valued_basic_functions.Rmd

```diff
@@ -127,6 +127,160 @@ with the same number of elements as `n2`.  For example,
 It would be illegal to try to assign `exp(n1)` to an array of
 integers; the return type is a real array.
 
+### Binary Function Vectorization
+
+Like the unary functions, many of Stan's binary functions have been
+vectorized, and can be applied elementwise to combinations of both
+scalars or container types.
+
+#### Real and real array arguments
+
+When applied to two real values, the result is a real value.  When
+applied to two arrays, or combination of a real value and an array,
```

Comment:
Maybe use scalar here instead of real value. I think some are also defined for int right?

Reply:
Good catch! Just updated those

Reply:
Thanks!